FND/OwnershipAndControl/Ownership.rdf

```diff
@@ -126,13 +126,13 @@
 		<rdfs:subClassOf>
 			<owl:Restriction>
 				<owl:onProperty rdf:resource="&fibo-fnd-oac-own;hasOwnedAsset"/>
```

Comment:
this seems to break the rule of including the range in the property name

Reply:
> this seems to break the rule of including the range in the property name

The other option was hasOwnedThingInRole, which seemed more complicated than it needed to be.  Naming is not easy, and for a complex pattern like this, it's even more difficult to be clear about what you are saying.  If you have a suggestion, please say so.  The current pattern is available in the EDW slides, attached, at the end of the presentation.
[EDW2021-ModelingPatterns-v6.pdf](https://github.com/edmcouncil/fibo/files/6368571/EDW2021-ModelingPatterns-v6.pdf)